### PR TITLE
Geoip 

### PR DESCRIPTION
**Pull request by registergoofy**
take care of the new geoip collection

### CODE REVIEW

File: wizard.sh [hunk 1]
```diff
@@ -388,6 +388,27 @@ update_bins() {
     systemctl restart crowdsec
 }
 
+geoip_workaround() {
```
Comment (erenJag) [inline wizard.sh:391]: not very clear as a function name.

File: wizard.sh [hunk 2]
```diff
@@ -388,6 +388,27 @@ update_bins() {
...
+    OUTPUT=$(${CSCLI_BIN_INSTALLED} collections list )
+    if printf '%s' "${OUTPUT}" | grep -q geoip; then
+	    log_info "Prout "
```
Comment (erenJag) [inline wizard.sh:394]: review the logging content please :)

File: wizard.sh [hunk 3]
```diff
@@ -388,6 +388,27 @@ update_bins() {
...
+	return
+    fi
+    OUTPUT=$(${CSCLI_BIN_INSTALLED}  collections list -o raw)
+    if  printf '%s' "${OUTPUT}" |grep -q "linux.yaml" ; then
+	PATH=$(${CSCLI_BIN_INSTALLED}  collections list)|awk -F" " '{print $6}'
+	if grep -q $PATH geoip ; then
+	    log_info "Prout 3"
```
Comment (erenJag) [inline wizard.sh:401]: here too

File: wizard.sh [hunk 4]
```diff
@@ -388,6 +388,27 @@ update_bins() {
...
+	    return
+	else
+	    log_info "Prout 4"
```
Comment (erenJag) [inline wizard.sh:404]: Same here

**Review by erenJag — CHANGES_REQUESTED** [review]

**Comment by buixor** [comment]
deprecated, we're going to issue debian-specific patch to deal with this

**Pull request closed without merging**